README.md

```diff
@@ -137,3 +137,33 @@ In particular this means that this project may not depend on GPL-licensed or AGP
 as these would violate the terms of those libraries' licenses.
 
 The contents of this repository are protected by Crown Copyright (C).
+
+## Performance
+The performance of PS Adaptor was tested with JMeter tool. The use case that was used was the simulation of the patient transfer request
```

Comment:
"the use case that was used..." reads a bit strangely, maybe change to "The use case tested was the...."

Reply:
thanks @MartinWheelerMT , sounds better indeed...